modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsFileDownloadFieldMerge.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Drupal\az_migration\Plugin\migrate\process;
+
+use Drupal\migrate\MigrateExecutableInterface;
+use Drupal\migrate\ProcessPluginBase;
+use Drupal\migrate\Row;
+use Drupal\media\Entity\Media;
+
+/**
+ * Configure Behavior for paragraphs.
+ *
+ * @MigrateProcessPlugin(
+ *   id = "paragraphs_file_download_field_merge"
+ * )
+ */
+class ParagraphsFileDownloadFieldMerge extends ProcessPluginBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function transform($value, MigrateExecutableInterface $migrate_executable, Row $row, $destination_property) {
+    // Merging the data into one field.
+    $field_uaqs_download_name = $row->getSourceProperty('field_uaqs_download_name');
+    $field_uaqs_download_description = $row->getSourceProperty('field_uaqs_download_description');
+
+    $value['uaqs_text'] = '<div class="border-thick border-top border-azurite"><div class="border card-body">
```

Comment:
I don't know if @danahertzberg wanted this wrapping divs added to the file download paragraphs (I think maybe just the extra info paragraphs).  Any thoughts @danahertzberg ?

Reply:
Removing from this migration would be good

Reply:
Fixed it!